### ISSUE TITLE
[Error: Attempt to invoke interface method 'void woyou.aidlservice.jiuiv5.IWoyouService.setAlignment(int, woyou.aidlservice.jiuiv5.ICallback)' on a null object reference]

### ISSUE THREAD

**Issue by ebrahim5801** [syumai/sunmi-namecard-printer#2]
![image](https://user-images.githubusercontent.com/116509746/223164480-a35ed290-a489-426d-bf97-35e5a8e0faf0.png)

**Comment by tarushbalodhi** [comment]
has this issue been solved ?

**Comment by ebrahim5801** [comment]
> has this issue been solved ?

No, I think the library works only with an older version of React Native

**Comment by tarushbalodhi** [comment]
I have fixed it by adding
<queries> <package android:name=“woyou.aidlservice.jiuiv5”/> </queries> 
in AndroidManifest.xml according to [https://developer.sunmi.com/docs/en-US/xeghjk491/ciqeghjk513](https://github.com/andrey-ushakov/flutter_sunmi_printer/issues/url)

**Comment by tarushbalodhi** [comment]
<queries> <package android:name=“woyou.aidlservice.jiuiv5”/> </queries>

**Comment by tarushbalodhi** [comment]
check this link 
https://github.com/andrey-ushakov/flutter_sunmi_printer/issues/4

**Comment by ebrahim5801** [comment]
> check this link [andrey-ushakov/flutter_sunmi_printer#4](https://github.com/andrey-ushakov/flutter_sunmi_printer/issues/4)

this was 2 years ago.
I tried it with an older version of react native and it worked, but I didn't use it because I had to use an old version of other libraries so we made the project using Flutter,
But if you know Java you can work your way around it, because after a lot of searching and debugging and trying, I couldn't fix it, you have to make updates in Java.
Good Luck.
